pkgs/development/python-modules/bootstrapped-pip/default.nix

```diff
@@ -42,7 +42,7 @@ in stdenv.mkDerivation rec {
   installPhase = ''
 
     # install pip binary
-    echo '#!${python.interpreter}' > $out/bin/pip
+    echo '#!${python.pythonForBuild.interpreter}' > $out/bin/pip
```

Comment:
Is this correct? Would this not put the wrong python into pip?

Reply:
Yeah this looks like an overbite. Maybe a this a holdover from `crossPython` being renamed?